source/common/common/to_lower_table.cc

```diff
@@ -0,0 +1,16 @@
+#include "common/common/to_lower_table.h"
+
+ToLowerTable::ToLowerTable() {
```

Comment:
I'm not super familiar with constexpr, but I tried it locally, and it doesn't like the for loop. I'm also not sure it would like the writes.